Gems/Atom/RHI/Vulkan/Code/CMakeLists.txt

```diff
@@ -68,7 +69,7 @@ if(NOT PAL_TRAIT_ATOM_RHI_VULKAN_SUPPORTED)
                     Gem::Atom_RHI.Edit
                     Gem::Atom_RHI.Reflect
                     3rdParty::glad_vulkan
-
+                    3rdParty::amd_vma
```

Comment:
Dont think we will need to add vma for Mac.

Reply:
Removed vma from the stubbed gems.